src/Header/ErrorHandle.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_ErrorHandle_H_
+#define GROUP4_HEADER_ErrorHandle_H_
+
+#include <string>
+
+// write error message to local file
+class ErrorHandle
```

Comment:
Consider moving other logging code into this class, and call this class LogHandle or similar.

Reply:
done.